MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+recursive-include examples *
```

Comment:
This file isn't part of the library's tooling/repo

Reply:
MANIFEST.in deleted

Reply:
Wow that was a quick edit :+1: